documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -61,3 +61,21 @@ To entirely ignore the display when connected, add the following line to `/boot/
 ----
 ignore_lcd=1
 ----
+
+== Attaching the Touch Display 2 LCD panel
+
+Touch Display 2 is a 720x1280 7" LCD display designed specifically for Raspberry Pi devices (see https://www.raspberrypi.com/products/touch-display-2/). It connects in much the same way as the original touch display, but the software setup on Compute Modules is slightly different as it uses a different display driver. See xref:../accessories/touch-display-2.adoc[Touch Display 2] for connection details.
```

Comment:
"It connects in much the same way as the original touch display"

Are there any differences? If there are, then what?

Reply:
Good point!

Reply:
Well, the power-supply leads are _slightly_ different in that they were two separate jumper wires on TD1, but a 1x3 block on TD2 :wink: